crates/bevy_gizmos/src/arcs.rs

```diff
@@ -36,29 +36,27 @@ where
     /// # use std::f32::consts::PI;
     /// # use bevy_color::palettes::basic::{GREEN, RED};
     /// fn system(mut gizmos: Gizmos) {
-    ///     gizmos.arc_2d(Vec2::ZERO, 0., PI / 4., 1., GREEN);
+    ///     gizmos.arc_2d(Isometry2d::IDENTITY, Rot2::FRAC_PI_4, 1., GREEN);
     ///
     ///     // Arcs have 32 line-segments by default.
     ///     // You may want to increase this for larger arcs.
     ///     gizmos
-    ///         .arc_2d(Vec2::ZERO, 0., PI / 4., 5., RED)
+    ///         .arc_2d(Isometry2d::IDENTITY, Rot2::FRAC_PI_4, 5., RED)
     ///         .resolution(64);
     /// }
     /// # bevy_ecs::system::assert_is_system(system);
     /// ```
     #[inline]
     pub fn arc_2d(
         &mut self,
-        position: Vec2,
-        direction_angle: f32,
-        arc_angle: f32,
+        isometry: Isometry2d,
+        arc_angle: Rot2,
```

Comment:
I feel like the arc angle should still be an `f32`. It's not really a rotation conceptually, so I think steering clear of `Rot2` is the right choice in this case. (The documentation also says it's an angle in radians, which is not really what a `Rot2` is.)

Reply:
Changed it. I mainly saw it as an extension of the API since users would be able to specify the angle in degrees as well. I see what you mean though and agree!